internal/testutil/acc/advanced_cluster.go

```diff
@@ -101,3 +103,29 @@ func CheckExistsClusterHandlingRetry(projectID, clusterName string) error {
 		return nil
 	})
 }
+
+func ConvertToTPFAttrsMap(attrsMap map[string]string) {
+	if !config.AdvancedClusterV2Schema() {
+		return
+	}
+	names := make([]string, 0, len(attrsMap))
+	for name := range attrsMap {
+		names = append(names, name)
+	}
+	for _, name := range names {
+		newName := name
+		for k, v := range tpfAttrMapping {
+			newName = strings.ReplaceAll(newName, k, v)
+		}
+		if newName != name {
+			attrsMap[newName] = attrsMap[name]
+			delete(attrsMap, name)
+		}
+	}
+}
+
+var tpfAttrMapping = map[string]string{
```

Comment:
nice

Reply:
went though another refactor, recommend to look at the latest version